lib/ruby_indexer/lib/ruby_indexer/rbs_indexer.rb

```diff
@@ -95,5 +105,38 @@ def add_declaration_mixins_to_entry(declaration, entry)
         entry.mixin_operations << mixin_operation if mixin_operation
       end
     end
+
+    sig { params(member: RBS::AST::Members::MethodDefinition, class_entry: T.any(Entry::Class, Entry::Module)).void }
```

Comment:
- The type should be `Entry::Namespace`
- `class_entry` should be renamed if we're taking module too